Dockerfile

```diff
@@ -171,13 +168,19 @@ RUN pip install --no-cache-dir "templateflow>=0.1.3,<0.2.0a0" && \
     find $TEMPLATEFLOW_HOME -type d -exec chmod go=u {} + && \
     find $TEMPLATEFLOW_HOME -type f -exec chmod go=u {} +
 
+# Installing bids-validator
+RUN npm install -g bids-validator@1.2.3
```

Comment:
I did this because this is prone to relatively frequent updates. For small things like this, AROMA, TemplateFlow, I'm inclined to move them to just before fMRIPrep as they get upgrades, which will ensure that the most frequently changed tend to be latest.